api.planx.uk/modules/send/bops/bops.test.ts

```diff
@@ -119,3 +125,102 @@ describe(`sending an application to BOPS`, () => {
       });
   });
 });
+
+describe(`sending an application to BOPS v2`, () => {
+  beforeEach(() => {
+    queryMock.mockQuery({
+      name: "FindApplication",
+      data: {
+        bopsApplications: [],
+      },
+      variables: {
+        session_id: "123",
+        search_string: "%/api/v2/planning_applications",
+      },
+    });
+
+    queryMock.mockQuery({
+      name: "CreateBopsApplication",
+      matchOnVariables: false,
+      data: {
+        insertBopsApplication: { id: 22 },
+      },
+    });
+  });
+
+  it.todo("throws an error if payload is invalid");
+
+  it.skip("proxies request and returns hasura id", async () => {
```

Comment:
Where you hitting any issues with this test, or where you just waiting to see what the BOPS response to this should be?

Reply:
This test is timing out currently! It's posting an empty string/invalid payload, which I'd expect to quickly throw an error _on our side before_ getting to the POST to BOPS step - so I'm planning to investigate if a) I'm missing a mock somewhere for the `makeDigitalPlanningPayload` method in planx-core or b) errors are failing to bubble up correctly.